src/components/Pagination/index.jsx

```diff
@@ -0,0 +1,112 @@
+import React from 'react';
+import { connect } from 'react-redux';
+import PropTypes from 'prop-types';
+import ReactPaginate from 'react-paginate';
+
+import classNames from 'classnames';
+import paginationStyles from './Pagination.scss';
+
+import { pageUpdate } from '../../data/actions/assets';
+
+export class Pagination extends React.Component {
+  constructor(props) {
+    super(props);
+    this.onPageClick = this.onPageClick.bind(this);
+  }
+
+  onPageClick(pageData) {
+    this.props.updatePage(pageData.selected);
+  }
+
+  getDisabledScreenReaderText() {
+    return (
+      <span className={paginationStyles['sr-only']}> button disabled</span>
+    );
+  }
+
+  getPreviousLabel(totalPages) {
+    return (
+      <span>
+      previous
+        {(this.props.assetsListMetaData.page === 0 && totalPages > 0) &&
```

Comment:
Actually, I believe that if there are 0 pages, the previous button is still marked as disabled by the Pagination component, so I may have to change this to >= 0. I'll have to verify.